benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -135,10 +135,10 @@ def time_less_than_scalar2(self, dtype):
 
 
 class CustomScalarFloorDivideInt(Benchmark):
-    params = ([np.int8, np.int16, np.int32, np.int64], [8, -8, 43, -43, 0])
+    params = ([*np.sctypes['int'], *np.sctypes['uint']], [8, -8, 43, -43, 0])
     param_names = ['dtype', 'divisors']
-    max_value = 10**7
-    min_value = -10**7
+    max_value = 10**2
+    min_value = -10**2
```

Comment:
How come this change now?

Reply:
I noticed in my previous PR, that small values itself show the changes properly, so I reduced it to make it memory and time friendly. I can make it 7 also, please let me know